packages/block-editor/src/components/font-sizes/font-size-picker.js

```diff
@@ -6,17 +6,19 @@ import { FontSizePicker as BaseFontSizePicker } from '@wordpress/components';
 /**
  * Internal dependencies
  */
-import useSetting from '../use-setting';
+import { useSettings } from '../use-setting';
 
 function FontSizePicker( props ) {
-	const fontSizes = useSetting( 'typography.fontSizes' );
-	const disableCustomFontSizes = ! useSetting( 'typography.customFontSize' );
+	const [ fontSizes, customFontSizes ] = useSettings( [
```

Comment:
Maybe we should keep the same name as the subsetting name:

```suggestion
	const [ fontSizes, customFontSize ] = useSettings( [
```

Reply:
Here I chose the variable name based on the name of the setting: `customFontSize`.

Reply:
I may be missing something, but that's exactly why I added the suggestion - in the PR we're using `customFontSizes` and not `customFontSize`.

Reply:
Ah, I see 🙂 Changed as suggested.